Gems/ScriptCanvas/Code/Editor/Components/EditorScriptCanvasComponent.cpp

```diff
@@ -30,481 +31,102 @@
 #include <ScriptCanvas/Components/EditorGraphVariableManagerComponent.h>
 #include <ScriptCanvas/Components/EditorScriptCanvasComponent.h>
 #include <ScriptCanvas/Components/EditorUtils.h>
-#include <ScriptCanvas/Core/Node.h>
-#include <ScriptCanvas/PerformanceStatisticsBus.h>
-
-namespace EditorScriptCanvasComponentCpp
-{
-    enum Version
-    {
-        PrefabIntegration = 10,
-        InternalDev,
-        AddSourceHandle,
-        RefactorAssets,
-        RemoveRuntimeData,
-        // add description above
-        Current
-    };
-}
+#include <ScriptCanvas/Execution/RuntimeComponent.h>
 
 namespace ScriptCanvasEditor
 {
-    static bool EditorScriptCanvasComponentVersionConverter(AZ::SerializeContext& serializeContext, AZ::SerializeContext::DataElementNode& rootElement)
```

Comment:
All moved deprecation data for objectstream data has been moved to a consolidated location for ease of future deprecation.